### DISCUSSION TITLE
Could not find a version that satisfies the requirement torch (from easyocr)

### DISCUSSION THREAD

**Discussion by ShivamS1234** [flet-dev/flet#4577] [Packages] [answered]
Hi @FeodorFitsner,

please add this package easyocr [https://pypi.org/project/easyocr/#files]. 

I am getting this issue in the flutter iOS. please help me.

Could not find a version that satisfies the requirement torch (from easyocr). I have package dependency of easyocr.

CMD: dart run serious_python:main package app/src -p iOS --requirements -r,app/src/requirements.txt

Requirements.txt:
easyocr==1.7.2

Output:

Building package executable...
Built serious_python:main.
Running package command
Extra PyPi indexes: [https://pypi.flet.dev]
Created temp directory: /var/folders/bl/6xsl0d397mxgv1_td1yx0t6m0000gq/T/serious_python_tempRoiF1z
Copying Python app from /Users/shivam.singh/Downloads/serious-python-main/src/serious_python/example/run_example/app/src to a temp directory
Configured iOS/iphoneos.arm64 platform with sitecustomize.py
Installing [-r, app/src/requirements.txt] with pip command to /Users/shivam.singh/Downloads/serious-python-main/src/serious_python/example/run_example/build/site-packages/iphoneos.arm64
ERROR: Could not find a version that satisfies the requirement torch (from easyocr) (from versions: none)
ERROR: No matching distribution found for torch

Thanks.

**Comment by ndonkoHenri** [comment, marked as answer]
The blocker here is `easyocr`'s hard dependency on PyTorch, which can't be packaged for Flet mobile apps (see https://github.com/flet-dev/flet/discussions/3245#discussioncomment-17512777). Since easyocr runs its detection/recognition models on torch, it can't work on-device as-is.
